learnbot_dsl/blocksConfig/basic/Control.conf

```diff
@@ -72,7 +72,7 @@
     ],
     "shape": ["block4", "block3"],
     "languages": { "ES": "tiempo_transcurrido", "EN": "elapsed_Time" },
-    "tooltip": { "ES": "Devuelve verdadero si el tiempo total del programa es mayor o igual que threshold sino devuelve falso", "EN": ""}
+    "tooltip": { "ES": "Devuelve verdadero si el tiempo total del programa es mayor o igual que threshold sino devuelve falso", "EN": "Returns true if the total time of the program is greater than or equal to threshold otherwise returns false"}
```

Comment:
```suggestion
    "tooltip": { "ES": "Devuelve verdadero si el tiempo total del programa es mayor o igual que 'threshold', si no devuelve falso", "EN": "Returns true if the total time of the program is greater or equal than 'threshold', otherwise it returns false"}
```